src/main/java/app/studnicki/ox/Main.java

```diff
@@ -0,0 +1,37 @@
+package app.studnicki.ox;
+
+class Main {
+  public static void main(String[] args) {
+    ConsoleUserInterface ui = new ConsoleUserInterface(System.in);
+
+    int n;
+
+    if (args.length > 0) {
+      try {
+        n = Integer.parseInt(args[0]);
+      } catch (NumberFormatException ex) {
+        ui.error(ex.getMessage());
+        n = 3;
+      }
+    } else {
+      n = 3;
+    }
+
+    Round round = new Round(n);
+
+    round.addObserver(ui);
+
+    ui.welcome();
+
+    ui.board(round.dimension, round.board);
+
+    while (round.board.size() < (round.dimension * round.dimension)) {
+      int id = ui.fieldId(round.dimension * round.dimension);
```

Comment:
maybe round.dimension*round.dmension can be a variable an you can use it in this to lines

Reply:
Yeah, I know. It is temporary. I think I should create Board class with limit variable within.